src/mparticle-instance-manager.ts

```diff
@@ -77,9 +77,15 @@ function mParticleInstanceManager(this: IMParticleInstanceManager) {
             self._instances[instanceName] = client;
         }
 
+        client.captureTimings(Constants.PerformanceMetricsNames.SdkStart);
+        client.setLauncherInstanceGuid();
```

Comment:
This implies that when app.js leads, that mparticle.js will load at the top of the file, and then launcher.js loads below when mParticle.calls init. Is that correct? Otherwise, the launcher will have its own id and this would never set it, right?